meshpy/element_beam.py

```diff
@@ -57,14 +56,14 @@ class Beam(Element):
     #        [ xi, is_middle_node ], # fist node
     #        ...
     #        ]
-    nodes_create = []
+    nodes_create: Any = []
 
     # A list of valid material types for this element.
-    valid_material = []
+    valid_material: Any = []
 
     # Coupling strings.
-    coupling_fix_string = None
-    coupling_joint_string = None
```

Comment:
This should stay at `None`, it is later on checked if it is None and an error is thrown (in `get_coupling_string`). I added an `Optional[str]` type hint.